src/DAL/Order/Cart.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DAL.Order
+{
+	public sealed class Cart
+	{
+		public Guid Id { get; set; }
+		public int UserId { get; set; }
+		public User User { get; set; }
```

Comment:
How do you think, wouldn't it be good to have CartItems here?

Reply:
Sure